test/CMakeLists.txt

```diff
@@ -203,9 +203,17 @@ foreach(SDK ${SWIFT_SDKS})
       list(APPEND test_dependencies "touch-covering-tests")
     endif()
 
+    # In Darwin, we depend on the fat targets.
+    is_darwin_based_sdk("${sdk}" IS_DARWIN)
+    if(IS_DARWIN)
```

Comment:
I think that we can use `if(sdk IN_LIST SWIFT_APPLE_PLATFORMS)` instead of the `is_darwin_based_sdk`.